tests/unit/test_index_dynamic_pagination.py

```
@@ -0,0 +1,560 @@
+        )
+        return requested
+
+    return _install
+
+
+def _run_sequential(end_page=2, start_page=1, **overrides):
+    kwargs = dict(
+        runtime=None, session=None, start_page=start_page, end_page=end_page,
+        parse_all=False, phase_mode='1', custom_url=None,
+        ignore_release_date=False, use_proxy=False, use_cf_bypass=False,
+        max_consecutive_empty=3, output_csv='out.csv', output_dated_dir='/tmp',
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)